File: java-advent-of-code/src/test/java/advent/twenty_eighteen/Day16ChronalClassificationTest.java
package advent.twenty_eighteen;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day16ChronalClassificationTest {
    @Test
    public void checkReal() {
        Day16ChronalClassification cc = new Day16ChronalClassification("/twenty_eighteen/Day16-ChronalClassification-input.txt");

        assertEquals(570, cc.getPart1Answer().intValue());
        assertEquals(503, cc.getPart2Answer().intValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_eighteen/Day16ChronalClassificationTest.scala
package advent.twenty_eighteen

import org.scalatest.{FlatSpec, Matchers}

class Day16ChronalClassificationTest  extends FlatSpec with Matchers {
  "2018 Day 16 - Input" should "calculate answers" in {
    val dp = new Day16ChronalClassification("/twenty_eighteen/Day16-ChronalClassification-input.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }

  "2015 Day 9 - Example #1" should "calculate answers" in {
    val dp = new Day16ChronalClassification("/twenty_eighteen/Day16-ChronalClassification-example#1.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }
}
